python/ray/data/dataset.py

```diff
@@ -1754,8 +1792,9 @@ def limit(self, limit: int) -> "Dataset[T]":
 
         Examples:
             >>> import ray
-            >>> ds = ray.data.range(1000) # doctest: +SKIP
-            >>> ds.limit(100).map(lambda x: x * 2).take() # doctest: +SKIP
+            >>> ds = ray.data.range(1000)
+            >>> ds.limit(100).map(lambda x: x * 2).take()
+            [0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 34, 36, 38]
```

Comment:
This clearly seems to be a bug.

@clarkzinzow Do you know what is going wrong here?

Reply:
Actually, this is expected -- take has a parameter `limit=20` that limits the number of elements that are taken to 20 by default.